docs/experiment/general/data-model.md

```diff
@@ -12,23 +12,23 @@ At the top level in Amplitude is your **organization**. Within an organization,
 
 ## Projects
 
-Experiment uses the same projects which are required for Amplitude Analytics. Generally speaking you want to create a project per product and per environment. Since [flags](#flags-and-experiments), [experiments](#flags-and-experiments), and [deployments](#deployments) only exist within a single project, you will want to duplicate these objects across projects for the same product.
+Like Amplitude Analytics, Experiment uses projects. As a best practice, create a project per product and per environment. Because [flags](#flags-and-experiments), [experiments](#flags-and-experiments), and [deployments](#deployments) only exist within a single project, duplicate these objects across projects for the same product.
 
 !!!tip "Copying a flag to another project"
     When developing a new feature with an experiment, you can create the experiment in the dev environment project to develop and test that the implementation is correct, then copy the experiment into the prod project to run the experiment in prod.
 
 ## Deployments
 
-In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Deployments have an associated randomly generated **deployment key** (a.k.a API key) which is used to uniquely identify the deployment and authorize requests to Amplitude Experiment's evaluation servers. 
+In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Deployments have an associated randomly generated **deployment key** (also known as an API key) which is used to identify the deployment and authorize requests to Amplitude Experiment's evaluation servers. 
 
 !!!info "Client vs Server Deployments"
-    Deployments are either client or server deployments. Only server deployments can utilize access flag configs for [local evaluation](./evaluation/local-evaluation.md), and should not be shared to made public in any way.
+    Deployments are either client or server deployments. Only server deployments can use access flag configs for [local evaluation](./evaluation/local-evaluation.md), and shouldn't be shared to made public in any way.
```

Comment:
@bgiori : >Only server deployments can use access flag configs for [local evaluation](./evaluation/local-evaluation.md), and shouldn't be shared to made public in any way.<

What's not meant to be shared or made public here? The deployment key or 'access flag configs' or just 'flag configs'?

Reply:
Just "flag configs", original sentence had a typo